### PR TITLE
Updated content for Contact the Microsoft Edge DevTools team

### PR DESCRIPTION
**Pull request by captainbrosset**
**Rendered page for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/contact?branch=pr-en-us-1788

I re-ordered, clarified, and updated the "Contact the Microsoft Edge DevTools team" page since we have a new way of contacting the team. These are the 2 ways to get in touch now:

1. the feedback button is great as it's in the tools, so can be quickly used to report a bug while it's happening
2. the new repo is coming and will allow for higher-quality conversations about new features, experiments, questions, etc.

I've tried to capture this way of thinking in the PR.

I've removed twitter as the team's account isn't very active and not a particularly better way to contact the team than the above 2 ways.

I've kept the WWW initiative and docs bugs reports, since those still make sense.

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/contact.md [hunk 1]
```diff
@@ -6,39 +6,40 @@ ms.author: msedgedevrel
...
 
-Send your feedback to help improve Microsoft Edge DevTools.  Contact the Microsoft Edge DevTools team to submit feature requests or bug reports of issues about developing for Microsoft Edge.
+Send your feedback to help improve Microsoft Edge DevTools.  Contact the Microsoft Edge DevTools team to submit feature requests or bug reports, or ask questions about Microsoft Edge DevTools.
 
-There are several ways to get in touch with the Microsoft Edge DevTools team.
+There are several ways to get in touch with the Microsoft Edge DevTools team:
+
+* Using the **Send feedback** window to quickly report bugs while using DevTools.
+* Using the **DevTools** repo to submit feature requests, questions, and have longer conversations with the team.
+* Using **The Web We Want** initiative to submit cross-browser suggestions.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/contact.md:19]:
```suggestion
* Using the **Web We Want** initiative to submit cross-browser suggestions.
```

Comment (vchapel) [reply microsoft-edge/devtools-guide-chromium/contact.md:19]: And maybe all "ing" such as "using" to "use" and "clicking" to "click" ...

Comment (vchapel) [reply microsoft-edge/devtools-guide-chromium/contact.md:19]: Or, as suggested by Michael, start with the goal such as "To submit cross-browser suggestions, use the **Web We Want** initiative." and "To quickly report bugs from DevTools, use the **Sendfeedback** window."

File: microsoft-edge/devtools-guide-chromium/contact.md [hunk 2]
```diff
@@ -6,39 +6,40 @@ ms.author: msedgedevrel
...
+## Use the Send feedback window
 
-To report a bug in DevTools or to request a new feature, [open DevTools](index.md#open-devtools) and then select the **Send Feedback** icon.  Or, in DevTools, press `Alt`+`Shift`+`I` (Windows, Linux) or `Option`+`Shift`+`I` (macOS).  This approach creates a tracked issue for the Microsoft Edge development team.
+To report a bug or a feature request while you are using DevTools, click the **Send feedback** icon or press `Alt`+`Shift`+`I` (Windows, Linux) or `Option`+`Shift`+`I` (macOS).  This approach creates a tracked issue for the Microsoft Edge development team.
 
-:::image type="content" source="../media/devtools-send-feedback.msft.png" alt-text="The Send Feedback icon in Microsoft Edge DevTools." lightbox="../media/devtools-send-feedback.msft.png":::
+![Screenshot of Microsoft Edge, with DevTools open, and the Send feedback icon in DevTools' toolbar](media/devtools-send-feedback.png)
 
-Selecting the **Send Feedback** icon opens a **Send Feedback** window.
+Clicking the **Send feedback** icon opens a **Send feedback** window which you can use to describe the bug you found or the feature request you have.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/contact.md:30]: To open a window where you describe the bug or feature request, click **Send feedback**.

File: microsoft-edge/devtools-guide-chromium/contact.md [hunk 3]
```diff
@@ -6,39 +6,40 @@ ms.author: msedgedevrel
...
+## Use the DevTools repo
 
+To have longer discussions with the DevTools team and community about bugs or features, use the DevTools repository on GitHub.
 
-<!-- ====================================================================== -->
-## Send a tweet to EdgeDevTools
+You can [create a new issue](https://github.com/MicrosoftEdge/DevTools/issues/new/choose) on the repo or [search for existing issues](https://github.com/MicrosoftEdge/DevTools/issues) and join conversations.
 
-Microsoft Edge DevTools has an [EdgeDevTools Twitter account](https://twitter.com/EdgeDevTools) with news.  Regarding developing with DevTools in Microsoft Edge, you can send a tweet to [@EdgeDevTools](https://twitter.com/intent/tweet?text=@EdgeDevTools).
+Using the DevTools repo is useful to have longer conversations, especially when you need more than report a bug. On the repo, you can share ideas and tips with others, explain your workflows and use cases in details with the team, and generally have more meaningful interactions with the community.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/contact.md:42]:
```suggestion
Using the DevTools repo is useful to have longer conversations, especially when you need to do more than report a bug. In the repo, you can share ideas and tips with others, explain your workflows and use cases in detail with the team, and generally have more meaningful interactions with the community.
```

File: microsoft-edge/devtools-guide-chromium/contact.md (hunk 1 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/contact.md:19]: Sounds great. Will address in a new commit.

File: microsoft-edge/devtools-guide-chromium/contact.md (hunk 2 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/contact.md:30]: The paragraph before the image already says to click the send feedback button. So this feels a bit like a repetition. I know how to fix this: I can re-structure this as a bullet list (first click button, then enter info in window), and then remove the first image, and only have one that shows both the button to click and the feedback window to use.

File: microsoft-edge/devtools-guide-chromium/contact.md (hunk 3 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/contact.md:42]: Thanks! Addressed in my next commit.

File: microsoft-edge/devtools-guide-chromium/contact.md [hunk 4]
```diff
@@ -6,39 +6,40 @@ ms.author: msedgedevrel
...
 
-Send your feedback to help improve Microsoft Edge DevTools.  Contact the Microsoft Edge DevTools team to submit feature requests or bug reports of issues about developing for Microsoft Edge.
+Send your feedback to help improve Microsoft Edge DevTools.  Contact the Microsoft Edge DevTools team to submit feature requests or bug reports, or ask questions about Microsoft Edge DevTools.
 
-There are several ways to get in touch with the Microsoft Edge DevTools team.
+There are several ways to get in touch with the Microsoft Edge DevTools team:
 
+* To quickly report bugs from DevTools, use the **Send feedback** window.
+* To submit feature requests, questions, and have longer conversations with the team, use the **DevTools** repo.
+* To submit cross-browser suggestions, use **The Web Web Want** initiative.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/contact.md:19]:
```suggestion
* To submit cross-browser suggestions, use the **Web Web Want** initiative.
```

File: microsoft-edge/devtools-guide-chromium/contact.md [hunk 5]
```diff
@@ -6,39 +6,40 @@ ms.author: msedgedevrel
...
+## Use the DevTools repo
 
+To have longer discussions with the DevTools team and community about bugs or features, use the DevTools repository on GitHub.
 
-<!-- ====================================================================== -->
-## Send a tweet to EdgeDevTools
+You can [create a new issue](https://github.com/MicrosoftEdge/DevTools/issues/new/choose) on the repo or [search for existing issues](https://github.com/MicrosoftEdge/DevTools/issues) and join conversations.
 
-Microsoft Edge DevTools has an [EdgeDevTools Twitter account](https://twitter.com/EdgeDevTools) with news.  Regarding developing with DevTools in Microsoft Edge, you can send a tweet to [@EdgeDevTools](https://twitter.com/intent/tweet?text=@EdgeDevTools).
+Using the DevTools repo is useful to have longer conversations, especially when you need to do more than report a bug. In the repo, you can share ideas and tips with others, explain your workflows and use cases in detail with the team, and generally have more meaningful interactions with the community.
```
Comment (vchapel) [inline microsoft-edge/devtools-guide-chromium/contact.md:42]:
```suggestion
Use the DevTools repo to have longer conversations, especially when you need to do more than report a bug. In the repo, you can share ideas and tips with others, explain your workflows and use cases in detail with the team, and generally have more meaningful interactions with the community.
```

**Review by vchapel — APPROVED** [review]
LGTM

**Comment by captainbrosset** [comment]
> I've removed twitter as the team's account isn't very active and not a particularly better way to contact the team than the above 2 ways.

Chatted with the team offline about this, and we're all onboard to do this.

**Pull request merged**